### ISSUE TITLE
Question from 10/3 lecture 

### ISSUE THREAD

**Issue by katarinabraun** [katarinabraun/stats679#3]
@cecileane @coraallencoleman 

I am going through lecture notes from 10/3 from the **safe script options** section and I have run into a problem.  

I am not able to run the headtail.sh script from anywhere, despite having moved it into my ~/bin directory. I get an error like this ```-bash: headtail.sh: command not found``` 

Here are the steps I took: 
1. Created the headtail.sh script. 
2. Created a bin/ directory within my home directory. 
3. Moved the headtail.sh script into ~/bin. 
4. Add this line to my .bash_profile: ```export PATH="$PATH:Users/katbraun/bin"``` 
5. Closed and reopened the terminal. 

What am I doing wrong?  
Thanks for your help!

**Comment by cecileane** [comment]
what you did looks great, so let's troubleshoot.
- What do you see when you do `echo $PATH`? Do you see your home bin directory `~/bin`?
- What do you see when you do `ls ~/bin`? just to triple-check that your script is there.
- Did you add the "shebang" line in your `headtail.sh` script? It should come on the very first line and look like this: `#!/bin/bash`. It looks like a comment line, but as the first line and starting with `#!`, it tells the shell what's supposed to run the script.

**Comment by katarinabraun** [comment]
okay, thank you! 

- Yes, ```echo $PATH``` does return the home bin directory I created – ```Users/katbraun/bin```, along with some other ```bin``` directories, separated by a colon 
- Yes, I see ```headtail.sh``` from ```ls ~/bin```
- And yes, the shebang line, like this: ```#!/bin/bash```,  is the first line of my headtail.sh script 

It's a puzzle!

**Comment by katarinabraun** [comment]
I get ```/bin/bash```.

Apologies for the trouble. Would it be easier to troubleshoot after class when you can see my terminal?

**Comment by cecileane** [comment]
yes definitely, it would be easier to troubleshoot after class. One last thought: did you change the permission of the file, to make it executable by yourself? `chmod u+x filename`. I don't think it's the reason, but that would be necessary anyway.
